hmmlearn/base.py

```diff
@@ -405,6 +406,13 @@ def sample(self, n=1, random_state=None):
                 currstate, random_state=random_state))
 
         return np.array(obs), np.array(hidden_states, dtype=int)
+    
+    def _is_identifiable(self, means, n_components):
```

Comment:
I am not sure I get what you mean. Indeed not all subclasses need this method, but it is applied in EM  iteration to check if the parameters have converged or converged properly. The argument means is not means_prior in _init but an estimation after e step. The problem is that there is only one generic fit method in which EM steps are conducted and that's why I call  _is_identifiable  in fit method.  I am open to any suggestion though.